package.json

```diff
@@ -48,7 +48,7 @@
   "scripts": {
     "precommit": "standard --fix js/**/*.js",
     "pretest": "standard --fix js/**/*.js",
-    "test": "./tests/js.sh"
+    "test": "./tests/js/js.sh"
```

Comment:
Why the `js` subdir?

Reply:
I needed to add a solidity file and deployment yaml for the js tests. 
I put them in a separate directory so it's clear that they're for the js test only